src/styles/gallery.scss

```diff
@@ -0,0 +1,71 @@
+.gallery {
+
+  &__title {
+    margin-bottom: 40px;
+  }
+
+  &__set {
+    display: grid;
+    column-gap: 20px;
+    overflow: hidden;
+
+    &--top {
+      grid-template-columns: 2fr 1fr;
+
+      @media (max-width: $breakPoint) {
+        grid-template-columns: 1fr 1fr;
+      }
+    }
+
+    &--middle {
+      grid-template-columns: 1fr 1fr;
+    }
+
+    &--bottom {
+      grid-template-columns: 1fr 2fr;
+
+      @media (max-width: $breakPoint - 1px) {
+        grid-template-columns: 1fr 1fr;
+      }
+    }
+
+  }
+
+  &__img {
+    height: 256px;
+    width: 100%;
+    object-fit: cover;
+    border-radius: 8px;
+
+    @media (max-width: $breakPoint - 1px) {
+      height: 30vw;
+    }
+  }
+
+  &__info {
+    margin-bottom: 40px;
+  }
+
+  &__button {
+    display: block;
+    width: 188px;
+    height: $buttonHeight;
+    margin: 0 auto;
+    color: $mainColor;
+    font-weight: bold;
+    background: #fff;
+    border: 2px solid $mainColor;
+    border-radius: 8px;
+    cursor: $buttonCursor;
```

Comment:
there is no need to add this value to constants

Reply:
fixed everywhere, thanks)